hoot-core/src/main/cpp/hoot/core/elements/OsmMap.h

```diff
@@ -335,6 +338,8 @@ class OsmMap : public std::enable_shared_from_this<OsmMap>, public ElementProvid
 
   // useful during debugging
   QString _name;
+  /** List of source URLs of map data */
+  std::set<QString> _sources;
```

Comment:
Used a `std::set` instead of a `QSet` because it is guaranteed ordering with iterators (something `QSet` doesn't guarantee) and doesn't allow duplicates.